cucumber/protractor-chrome.config.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
```

Comment:
Could you please move the config files to the config folder?

Reply:
done